editor/StdAfx.h

```diff
@@ -1,7 +1,10 @@
 /*
-* Descent 3 
+* Descent 3
 * Copyright (C) 2024 Parallax Software
 *
+* Descent 3: Apex
```

Comment:
We probably don't want "Apex" mentions in here?

Reply:
It's code I ported from Apex, I did not feel like removing him copyright